python/cloudtik/runtime/metastore/scripts/configure.sh

```diff
@@ -28,7 +28,7 @@ do
     shift
 done
 
-if [ not $IS_HEAD_NODE == "true" ]; then
+if [ $IS_HEAD_NODE != "true" ]; then
```

Comment:
is "if [ not $IS_HEAD_NODE == "true" ]; then" working?

If not, we need to change also in install.sh for the same check.

Reply:
Use "if [ not $IS_HEAD_NODE == "true" ]"  will report error with "[: too many arguments" in our docker image, so I change it.